### ISSUE TITLE
docker compiled with tensorflow with AVX instruction  set

### ISSUE THREAD

**Issue by toni-moreno** [regel/loudml#106]
Hi @regel . I'm playing with loudml  and influxdb. When trying to save a mode. The docker image on the dockerhub site, give us this error.

````
loudml_1_bd75b481bef7 | INFO:root:starting Loud ML server on 0.0.0.0:8077
loudml_1_bd75b481bef7 | 2019-03-22 11:20:37.753162: F tensorflow/core/platform/cpu_feature_guard.cc:37] The TensorFlow library was compiled to use AVX instructions, but these aren't available on your machine.
````

I've seen that it has to be with the underlying infraestructure. ( https://github.com/tensorflow/tensorflow/issues/24548)  , but I'm working with a VM on a VMware host. 

 There is any way to get an loudml image without  AVX dependant tensorflow version ?

Thanks a lot!

**Comment by toni-moreno** [comment]
I don't know exactly why but after  create a image from source as described on https://github.com/regel/loudml/issues/107 the error dissapeared

**Comment by activiteOCR** [comment]
Hello @toni-moreno @regel , 
I'm creating my first model from Docs example and using Create model command.

`loudml create-model avg_temp2.json`

Here my json:

`{
  "bucket_interval": "1m",
  "default_datasource": "myinfluxdb",
  "features": [
    {
      "default": 0,
      "field": "temp2",
      "measurement": "temperature_series",
      "metric": "avg",
      "name": "avg_temp_feature"
    }
  ],
  "interval": 60,
  "max_evals": 10,
  "name": "avg_temp2-model",
  "offset": 30,
  "span": 5,
  "max_threshold": 90,
  "min_threshold": 50,
  "type": "donut"
}
`
I recovered an issue like you tensorflow/core/platform/cpu_feature_guard.cc:37] The TensorFlow library was compiled to use AVX instructions, but these aren't available on your machine.

I found this comment about the same error:

> The current version of Tensorflow installed via pip uses the AVX instruction set at compile time.This means that your CPU needs to support the AVX instruction set. This instruction set is supported from the second generation of Intel Core CPUs (codenamed SandyBridge). You can compile a Tensorflow from the source that does not use the AVX instruction set. Or find an already compiled one on the internet.

So right now I'm looking for any information to solve this issue.

I also use a virtual machine (virtualbox oracle) with debian stretch as distribution.
It seems that my computer doesn't support AVX instruction...
